integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -850,13 +908,17 @@ fn test_resharding_v3_split_parent_buffered_receipts() {
         .deploy_test_contract(receiver_account.clone())
         .limit_outgoing_gas()
         .add_loop_action(call_burn_gas_contract(
-            vec![account_in_left_child, account_in_right_child.clone()],
+            vec![account_in_left_child.clone(), account_in_right_child.clone()],
             receiver_account,
             10 * TGAS,
         ))
         .add_loop_action(check_receipts_presence_at_resharding_block(
             account_in_right_child,
             ReceiptKind::Buffered,
+        ))
+        .add_loop_action(check_receipts_presence_after_resharding_block(
+            account_in_left_child,
+            ReceiptKind::Buffered,
```

Comment:
Could you remind why we check right child at resharding block and left child after resharding block?

Reply:
before resharding it doesn't matter - they're both in the parent shard

after resharding the buffered receipts should only be present in the left child because that is the design for resharding buffered receipts - give them all to the left child